deprecating net5.0 azurescripting

IsaacCalligeros95 opened the pull request:
Upgrading net5.0 references to net6.0.

IsaacCalligeros95 committed:
upgrading from net5.0 to net6.0

mjhilton reviewed (changes requested):
Looks like you've accidentally checked in something from your local that wasn't intended

IsaacCalligeros95 commented:
> Looks like you've accidentally checked in something from your local that wasn't intended

I've removed the commented-out tests call and rebased, comments about the GetRSAPrivateKey() calls are on the AzureCloudService [PR](https://github.com/OctopusDeploy/Sashimi.AzureCloudService/pull/31/files).

mjhilton reviewed (approved):
LGTM
